File: src/backend/belsim2020/belsim2020.Entities/Project.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace belsim2020.Entities
{
	[Table("Projects")]
	public class Project
	{
		[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
		public Guid ProjectId { get; set; }

		public string ProjectName { get; set; }

		public DateTime CreatedAt { get; set; }
		public DateTime ModifiedAt { get; set; }

		public RkExperimentType ProjectType { get; set; }

		public virtual ICollection<UserProject> Users { get; set; }
	}
}

File: src/frontend/src/models/Project.ts
export enum BelsimProjectType {
    RK = '1'
}
export class Project {
    public projectId: string = '';
    public projectName: string = '';
    public createdAt: Date = new Date();
    public modifiedAt: Date = new Date();
    public projectType: BelsimProjectType = BelsimProjectType.RK;
    public organizationName: string = '';
    public comments: string = '';
}